### PR TITLE
Test on Chromium on Travis

### PR DESCRIPTION
**Pull request by ptitjes**
This PR adds
- a CHROME_BIN option to selenium test,
- test jobs on chromium

For instance, to run with Chromium, use:
```bash
CLIENT=selenium:chrome CHROME_BIN=/usr/bin/chromium-browser npm run test
```

### CODE REVIEW

**Comment by ptitjes** [comment]
Hmm, I don't get why this fails. I tested [here](https://travis-ci.org/pouchdb/pouchdb/jobs/458207282#L703) that `chromium` is indeed available. And I tested locally that the `CHROME_BIN` variable is correctly used by https://github.com/pouchdb/pouchdb/pull/7538/commits/64b23e476cd7401342b15f89dcd6681bfc57219b and correctly sent to the ChromeDriver. If anyone has a suggestion, that would be great... @daleharvey, any ideas ?

**Comment by ptitjes** [comment]
It seems the bugs that hit Chromium 69 with pouchdb-find have been fixed with Chromium 70.

So multiple questions, for @daleharvey and others:

- Does anyone know of anyway to install an old version of Chromium on Ubuntu based on its version ?
I googled but didn't find something satisfying enough to be integrated in Travis.

- Is it still of value to have tests running on Chromium ? If yes which ones ?
For now, I copied the ones run on `selenium:firefox` which makes 6 new jobs. Is it too much ?
I would say main tests, map/reduce and find is the minimum.

File: .travis.yml [hunk 1]
```diff
@@ -49,17 +49,24 @@ env:
   - CLIENT=selenium:firefox COMMAND=test
   - CLIENT=selenium:firefox POUCHDB_SRC="../../packages/node_modules/pouchdb/dist/pouchdb.min.js" COMMAND=test
 
+  # Test in chromium running on travis
+  - CLIENT=selenium:chrome CHROME_BIN=/usr/bin/chromium-browser COMMAND=test
+  - CLIENT=selenium:chrome CHROME_BIN=/usr/bin/chromium-browser POUCHDB_SRC="../../packages/node_modules/pouchdb/dist/pouchdb.min.js" COMMAND=test
```
Comment (daleharvey) [inline .travis.yml]: No need to test the .min, mostly there to make sure packaging doesnt break

File: .travis.yml [hunk 2]
```diff
@@ -49,17 +49,24 @@ env:
...
+
   # Test auto-compaction in Node and Firefox
   - AUTO_COMPACTION=true CLIENT=node COMMAND=test
   - AUTO_COMPACTION=true CLIENT=selenium:firefox COMMAND=test
+  - AUTO_COMPACTION=true CLIENT=selenium:chrome CHROME_BIN=/usr/bin/chromium-browser COMMAND=test
```
Comment (daleharvey) [inline .travis.yml]: Same with AUTO_COMPACTION

File: .travis.yml [hunk 3]
```diff
@@ -85,6 +92,7 @@ env:
 
   # Performance tests
   - CLIENT=selenium:firefox PERF=1 COMMAND=test
+  - CLIENT=selenium:chrome CHROME_BIN=/usr/bin/chromium-browser PERF=1 COMMAND=test
```
Comment (daleharvey) [inline .travis.yml]: And PERF

**Comment by ptitjes** [comment]
Arghh, I should have put the PR number at the front of the squashed commit message... Sorry!

**Pull request merged**